simtools/model/telescope_model.py

```diff
@@ -502,14 +519,35 @@ def exportConfigFile(self):
         if not self._isExportedModelFilesUpToDate:
             self.exportModelFiles()
 
-        self.printParameters()
-
         # Using SimtelConfigWriter to write the config file.
         self._loadSimtelConfigWriter()
         self.simtelConfigWriter.writeTelescopeConfigFile(
             configFilePath=self._configFilePath, parameters=self._parameters
         )
 
+    def exportDerivedFiles(self, fileNames):
+        """
+        Write to disk a file from the derived values DB.
+
+        Parameters
+        ----------
+        dest: str or Path
```

Comment:
Is `dest` a parameter?

Reply:
No, relic from previous version of the code.